src/core/ir/model.rs

```diff
@@ -42,6 +46,7 @@ pub enum IO {
         group_by: Option<GroupBy>,
         dl_id: Option<DataLoaderId>,
         http_filter: Option<HttpFilter>,
+        is_list: bool,
```

Comment:
I think `is_list` can be moved to `IO` so that we can use it with gRPC also.

Reply:
should be done separately as `is_list` is only used in the http and fixing this is not part of this pr

Reply:
https://github.com/tailcallhq/tailcall/issues/2891